src/components/ChartScale/ChartScale.css

```diff
@@ -27,13 +27,15 @@
 @media only screen and (min-width: 992px) {
     .chart-scale {
         display: inline-flex;
+        width: unset;
     }
 
     .chart-scale__label-container {
         display: flex;
         flex-direction: column-reverse;
         border-right: 1px solid #ccc;
```

Comment:
I think now with the yellow background, this grey is too light. Also can we get a little margin between the text and the axis?